### DISCUSSION TITLE
Property '$exists' does not exist on type

### DISCUSSION THREAD

**Discussion by ivenuss** [prisma/prisma#7225] [Q&A] [answered]
Hey, can I have a question, why am I unable to get `$exists` property of `prisma`?
This what documentation shows:
```
const userExists: boolean = prisma.$exists.user({
  id: 'cjli6tko8005t0a23fid7kke7',
})
```
My code:
![image](https://user-images.githubusercontent.com/43939822/119227413-01427180-bb0e-11eb-92b7-8c9adc4e6054.png)

**Comment by janpio** [comment]
Can you link to the documentation you are referring to? I think you might be lookign at Prisma 1 documentation, but you are trying to use Prisma 2.

**Reply by ivenuss** [reply]
Oh yeah, it's Prisma 1 doc =>  https://v1.prisma.io/docs/1.34/prisma-client/features/check-existence-TYPESCRIPT-pyl2/
How can I achieve the same result with Prisma 2?

**Reply by janpio** [reply]
Not sure we have a wrapper. You can always do a `findUnique` or `findFirst` and then see if you get a result or not.

**Comment by ivenuss** [comment, marked as answer]
Found **upsert** function that handles my problem.
https://www.prisma.io/docs/concepts/components/prisma-client/crud#update-or-create-records
